pkg/manager/impl/validation/node_execution_validator.go

```diff
@@ -20,6 +21,35 @@ func ValidateNodeExecutionIdentifier(identifier *core.NodeExecutionIdentifier) e
 	return ValidateWorkflowExecutionIdentifier(identifier.ExecutionId)
 }
 
+func ValidateNodeExecutionEventRequest(request *admin.NodeExecutionEventRequest) error {
```

Comment:
i know the file does not have it, but some documentation please?

Reply:
done